gradle-plugin/doc/src/main/asciidoc/inc/_groovy-configuration.adoc

```diff
@@ -0,0 +1,252 @@
+
+[[groovy-configuration]]
+= Groovy Configuration
+
+[[global-configuration]]
+== Global Configuration
+.Global configuration
+[cols="1,5,1"]
+|====
+| Element | Description | Property
+| *buildStrategy*
+a| Defines what build strategy to choose while building container image.
+Possible values are `docker` and `jib` out of which `docker` is default.
+
+ifeval::["{goal-prefix}" == "oc"]
+If the effective build mode is `openshift` an additional `s2i` option is available
+and selected by default.
+
+Available strategies for OpenShift are:
+
+* `s2i` for a https://docs.openshift.com/enterprise/latest/architecture/core_concepts/builds_and_image_streams.html#source-build[Source-to-Image build] with a binary source
+* `docker` for a https://docs.openshift.com/enterprise/latest/architecture/core_concepts/builds_and_image_streams.html#docker-build[Docker build] with a binary source
+
+endif::[]
+| `jkube.build.strategy`
```

Comment:
Right now these properties don't seem to be working. If I pass them in gradle goal like this, it has no effect:

`gradle -Djkube.build.strategy=docker k8sBuild`

I see it's not handled in KubernetesExtension. Do we want to support this or shall I remove this column from here?